otherlibs/win32unix/winwait.c

```diff
@@ -23,16 +23,16 @@
 
 static value alloc_process_status(HANDLE pid, int status)
 {
-  value res, st;
+  CAMLparam0();
+  CAMLlocal1(st);
+  value res;
 
   st = caml_alloc(1, 0);
   Field(st, 0) = Val_int(status);
```

Comment:
While we're here, this is shorter and (academically) slightly faster:
```suggestion
  st = caml_alloc_boxed(Val_int(status));
```

Reply:
Thanks, fixed.